tutorials/kubeadm.md

```diff
@@ -21,3 +21,42 @@ Note: This file assumes you've set your cgroup_driver as systemd
 
 Given you've set CIDR, and you've properly set the kubelet file, all you need to do is start crio (as defined [here](setup.md)), and run:
 `kubeadm init --pod-network-cidr=$CIDR`
+
+# Running kubeadm in a off line network
+
+We will assume that the user has installed CRI-O and alls necessary packages. We will also assume that all necessary components are configured and everything is working as expected. The user should have a private repo where the docker images are pushed. Sample of images fot version 1.18.2:
+
+```bash
+$ kubeadm config images list --image-repository user.private.repo --kubernetes-version=v1.18.2
+user.private.repo/kube-apiserver:v1.18.2
+user.private.repo/kube-controller-manager:v1.18.2
+user.private.repo/kube-scheduler:v1.18.2
+user.private.repo/kube-proxy:v1.18.2
+user.private.repo/pause:3.2
+user.private.repo/etcd:3.4.3-0
+user.private.repo/coredns:1.6.7
+```
+
+The user needs to configure the [registries.conf](https://www.mankier.com/5/containers-registries.conf) file.
+
+Sample of configurations:
+
+```bash
+$ cat /etc/containers/registries.conf
+[[registry]]
+prefix = "k8s.gcr.io/pause:3.2"
```

Comment:
We regex here so instead of one image you could cover all them in the example.

Reply:
The images differ from each other. I am not sure if one regex can match all in once. On the other hand I am not that good with regex. Can you propose a sample and we can take a look?